webextensions/manifest/optional_permissions.json

```
@@ -377,10 +355,11 @@
         },
         "dns": {
           "__compat": {
-            "description": "<code>dms</code>",
+            "description": "<code>dns</code>",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": true,
```

```suggestion
                "version_added": "preview",
```

Should we use "preview" instead? https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#choosing-preview-values